makefiles/pseudomodules.inc.mk

```diff
@@ -94,6 +94,8 @@ PSEUDOMODULES += od_string
 # include variants of the AT86RF2xx drivers as pseudo modules
 PSEUDOMODULES += at86rf23%
 PSEUDOMODULES += at86rf21%
+PSEUDOMODULES += at86rfa1
+PSEUDOMODULES += at86rfr2
```

Comment:
I would have no problem with just using `PSEUDOMODULES += at86rf%`. If a conflict arises in the future, this could be fine tuned again.

Reply:
Actually that will conflict with at86rf215 where I change `at86rf21%` -> `at86rf212b`, so changing it here just to change it back in another PR seems a bit pointless.

Reply:
I see